internal/commands/project.go

```diff
@@ -217,7 +216,7 @@ func updateProjectRequestValues(input *[]byte, cmd *cobra.Command) error {
 	if projectName != "" {
 		info["name"] = projectName
 	} else {
-		return errors.Errorf("Project name is required")
+		return errors.Errorf(clierrors.ProjectNameIsRequired)
```

Comment:
done

Reply:
Error: don't use MixedCaps in package name; errorConstants should be errorconstants (golint)
Also, cannot be changed to "errors" because of the standard errors library.
Changed to errorconsts